setup.py

```diff
@@ -48,7 +49,29 @@ def get_readme() -> str:
         return f.read()
 
 
+class InstallCommand(install):
+
+    def run(self):
+        """Run the install command."""
+        install.run(self)
+        from sys import platform
+        import warnings
+
+        if platform == "darwin":
+            warnings.warn(
+                "Installation of important packages for macOS is required. "
+                "Scripts in the experiments folder will likely not run without these packages: "
+                "gnu-getopt, parallel, coreutils. They can be installed with Homebrew by running "
+                "`brew install gnu-getopt parallel coreutils`."
+                "See https://brew.sh/ for installation instructions.",
+                category=Warning,
```

Comment:
`Warning` is the base category; is there something more specific we can use? The default of `warnings.warn` of `UserWarning` seems preferable but may not be the best option.

Reply:
We can create a custom class called `CompatibilityWarning` or something of the kind, or just use UserWarning. I don'
t see any other more fitting default class.

Reply:
`UserWarning` seems OK, making a new class is probably overkill